source/sdk/math.ooc

```diff
@@ -86,36 +86,31 @@ extend Int {
 	maximumValue ::= static INT_MAX
 	pi ::= static 3
 	e ::= static 2
-	// Static Utility Functions
-	// parse: static func(value: String) -> This { This parse(value, 0) }
-	// parse: static func(value: String, default: This) -> This { }
-//	toString: static func(value: This) -> String {
-//		value toString()
-//	}
-	absolute: static func(value: This) -> This {
+
+	absolute: static func (value: This) -> This {
 		value >= 0 ? value : -1 * value
 	}
-	sign: static func(value: This) -> This {
+	sign: static func (value: This) -> This {
 		value >= 0 ? 1 : -1
 	}
-	maximum: static func ~two(first: This, second: This) -> This {
+	maximum: static func ~two (first: This, second: This) -> This {
 		first > second ? first : second
 	}
 	// TODO: Avoid using this, consider removing it.
-	maximum: static func ~multiple(value: This, values: ...) -> This {
+	maximum: static func ~multiple (value: This, values: ...) -> This {
 		values each(|v|
-			if ((v as This*)@ > value)
+			if ((v as This *)@ > value)
 				value = v
 		)
 		value
 	}
-	minimum: static func ~two(first: This, second: This) -> This {
+	minimum: static func ~two (first: This, second: This) -> This {
 		first < second ? first : second
 	}
 	// TODO: Avoid using this, consider removing it.
-	minimum: static func ~multiple(value: This, values: ...) -> This {
+	minimum: static func ~multiple (value: This, values: ...) -> This {
 		values each(|v|
-			if ((v as This*)@ < value)
+			if ((v as This *)@ < value)
```

Comment:
And it's not a bug, it's just related to the fact that ```*``` is disregarded since it can mean different things, as we see here. The violated rule here is the one that deals with keyword spacing (in this case, ```This```). I've added ```*``` to the ignore list for this particular rule and should have a release up in a few minutes.